frontend/public/locales/en/translation.json

```diff
@@ -740,6 +740,8 @@
     "Control plane pods": "Control plane pods",
     "Control plane status": "Control plane status",
     "Control plane type": "Control plane type",
+    "Control plane type - AWS": "Control plane type - AWS",
```

Comment:
@o-farag For the translations, maybe you can use a variable instead for the type so we can reuse the same translation. Something like this https://github.com/fxiang1/console/blob/main/frontend/src/components/TemplateSummaryModal.tsx#L185

Reply:
Alright, changed it to use a variable instead.